src/CheckoutSdk.Extensions/CheckoutServiceCollection.cs

```diff
@@ -17,6 +17,29 @@ public static class CheckoutServiceCollection
         {
             CheckoutUtils.ValidateParams("serviceCollection", serviceCollection, "configuration", configuration);
             var checkoutOptions = configuration.GetCheckoutOptions();
+
+            return AddSdkFromOptions(serviceCollection, checkoutOptions, loggerFactory, httpClientFactory);
+        }
+
+        public static IServiceCollection AddCheckoutSdk(
+            this IServiceCollection serviceCollection,
+            IConfigurationSection configurationSection,
+            ILoggerFactory loggerFactory = null,
+            IHttpClientFactory httpClientFactory = null)
+        {
+            CheckoutUtils.ValidateParams(nameof(serviceCollection), serviceCollection,
```

Comment:
Not really sure if nameof is correct here, this can be  `CheckoutUtils.ValidateParams("serviceCollection", serviceCollection);` only

I think your assumption is that this param is never null right?